lib/PluginFindRecord/PluginFindRecordModal.js

```diff
@@ -0,0 +1,401 @@
+            initialSearch={initialSearch}
+            initialSearchState={{ qindex: '', query: '' }}
+            syncToLocationSearch={false}
+          >
+            {
+              ({
+                searchValue,
+                getSearchHandlers,
+                onSubmitSearch,
+                onSort,
+                getFilterHandlers,
+                activeFilters,
+                filterChanged,
+                searchChanged,
+                resetAll,
+              }) => {
+                const disableReset = () => {
+                  if (filterChanged || searchChanged) {
+                    return false;
+                  }
+
+                  return true;
+                };
+
+                return (
+                  <Paneset id={`${idPrefix}-paneset`}>
+                    {this.state.filterPaneIsVisible &&
+                      <Pane
+                        defaultWidth="22%"
+                        paneTitle={<FormattedMessage id="stripes-smart-components.searchAndFilter" />}
+                      >
+                        <form onSubmit={onSubmitSearch}>
+                          <div className={css.searchGroupWrap}>
+                            <SearchField
+                              autoFocus
+                              className={css.searchField}
+                              data-test-plugin-search-input
+                              marginBottom0
+                              name="query"
+                              onChange={getSearchHandlers().query}
+                              value={searchValue.query}
+                            />
+                            <Button
+                              buttonStyle="primary"
+                              data-test-plugin-search-submit
+                              disabled={(!searchValue.query || searchValue.query === '')}
+                              fullWidth
+                              marginBottom0
+                              type="submit"
+                            >
+                              <FormattedMessage id="stripes-smart-components.search" />
+                            </Button>
+                          </div>
+                          <div className={css.resetButtonWrap}>
+                            <Button
+                              buttonStyle="none"
+                              disabled={disableReset()}
+                              fullWidth
+                              id="clickable-reset-all"
+                              onClick={resetAll}
+                            >
+                              <Icon icon="times-circle-solid">
+                                <FormattedMessage id="stripes-smart-components.resetAll" />
+                              </Icon>
+                            </Button>
+                          </div>
+                          {renderFilters
+                            ? renderFilters(activeFilters.state, getFilterHandlers())
```

Comment:
It seems there isn't a way to get resources (such as `data` into `renderFilters`). Am I missing something? Eg, would I be able to [use a set of filters like this](https://github.com/folio-org/ui-plugin-find-agreement/blob/master/src/Filters.js) that uses the `data` prop filled out by the [Container](https://github.com/folio-org/ui-plugin-find-agreement/blob/master/src/Container.js)?

Reply:
Maybe I didn't catch the question, but you can pass whatever you want to this function in Container, it's a callback from there.

Reply:
🤦‍♂ You're right, of course. I'm struggling today.